dashboards/component/form.py

```diff
@@ -77,6 +80,13 @@ def get_form(self, request: HttpRequest = None) -> DashboardForm:
                 data = request.GET
 
         form = self.form(data=data)
+
+        # Create and apply the GenericFilter if filter_data and filter_fields are provided
+        if self.filter_data and self.filter_fields and self.model:
```

Comment:
I may be misunderstanding something but why are we applying a filter here, the form shouldn't be doing any filtering right? it will just be providing the ability to submit data?

Reply:
filtering removed from form